.vscode/mcp.json

```diff
@@ -0,0 +1,9 @@
+{
+	"servers": {
```

Comment:
Nit, could we stay consistent with 2 spaces for indent?

Reply:
updated